src/App.css

```diff
@@ -2,14 +2,22 @@
   font-family: 'Roboto', sans-serif;
 }
 
-.video-previews {
+.shorts-previews {
   display: flex;
   flex-wrap: wrap;
   max-width: 1880px;
   margin: 0 16px;
 }
 
-.shorts-previews {
+.movie-previews {
```

Comment:
can you make a class called `previews` and add it to both shorts/movie previews. something like:

```
.previews {
  display: flex;
  flex-wrap: wrap;
  max-width: 1880px;
  margin: 0 16px;
}

.movie-previews {
  margin-top: 40px;
}
``` 

Then in `App.tsx` you would do something like:
```
<div className="previews video-previews"> .. </div>
<div className="previews movie-previews"> .. </div>
<div className="previews shorts-previews"> .. </div>
```